backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -176,7 +181,7 @@ public class CsvValidatorUtils {
           "n", "no",
           "u", UNKNOWN_CODE);
   private static final Set<String> TEST_RESULT_VALUES =
-      Set.of("positive", "negative", "not detected", "detected", "invalid result");
+      Set.of(POSITIVE_LITERAL, "negative", "not detected", DETECTED_LITERAL, "invalid result");
```

Comment:
Unrelated to this change but I realized we need to update the view for the device code lookup because we put "Undetermined" as the test result in the device code lookup tool but we don't support that in the bulk upload and accept "Not detected" instead.

See: https://github.com/CDCgov/prime-simplereport/blob/db562d21c605a529b6d7b421bc77ad75803e70f3/frontend/src/app/uploads/DeviceLookup/DeviceDetails.tsx#L18

**Device code lookup**
<img width="705" alt="Screenshot 2024-06-14 at 16 08 20" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/95fd9961-2f41-4013-9252-74decb93cf0a">

Reply:
Nice find on this! Yeah I think this is definitely worth creating a new ticket to update the device code lookup, especially given the recent issues with unmapped SNOMED code descriptions in the bulk uploader. Plus noticing that what we have listed as "Positive" is actually the SNOMED code for "Detected" whereas "Positive" SNOMED is 10828004. Seems like we're moving in the direction where it's important that certain tests distinguish the particular test result codes they use, so might be something else that needs updating too

Reply:
Ok! Will make a ticket for this!

Reply:
Ticket created here: https://github.com/CDCgov/prime-simplereport/issues/7826 Feel free to add/edit anything as you see fit!